lib/api/api_auth_webauthn.go

```diff
@@ -477,6 +473,27 @@ func (s *webauthnService) storeVolatileState(state *WebauthnVolatileState) error
 	return s.miscDB.PutBytes(s.miscDBKey, stateBytes)
 }
 
+func (s *webauthnService) updateCredentialVolatileState(credId string, updatedCred *webauthnLib.Credential) {
+	var signCountBefore uint32 = 0
```

Comment:
Why not rely on the default zero value?

Reply:
No reason other than unfamiliarity with Go conventions. Would this be clearer without the explicit zero?

Reply:
Ok. Implicit type using `:=` doesn't work in this case because that infers the type as `int`, which then conflicts further down in the `signCountBefore = dynCredState.SignCount` statement.

Anyway, I'll remove the explicit zero then.